cran-comments.md

```diff
@@ -1,3 +1,12 @@
+## Resubmission
+
+This is a resubmission. In this version I have:
+* Expanded Descrption section of `DESCRIPTION` to further describe package and
+cite authors of previous packages
+* Added The Ottawa Hospital as author
+* Added executable examples to exported functions
+* Enabled automated testing of functions
```

Comment:
add thats its through the use of testthat. Also we added some additional features which can be found in the changelog. Perhaps reference the additional features and why we are also adding them and not just the fixes they requested

Reply:
I'd reference the changelog for 'other additional features not requested by CRAN review but arose during revisions'.